projects/plugins/jetpack/extensions/blocks/ai-paragraph/index.js

```diff
@@ -73,7 +73,7 @@ export const settings = {
 	edit,
 	save: attrs => {
 		const blockProps = useBlockProps.save();
-		return <RawHTML { ...blockProps }>{ attrs.attributes.content }</RawHTML>;
+		return <RichText.Content { ...blockProps } value={ attrs.attributes.content } />;
```

Comment:
I would wrap this in `p` tags
```suggestion
		return <p {...blockProps}><RichText.Content value={ attrs.attributes.content } /></p>;
```

Reply:
This should make the HTML look "better".